theme/views.py

```diff
@@ -106,6 +106,10 @@ def get_context_data(self, **kwargs):
                     Q(raccess__public=True) | Q(raccess__discoverable=True)
                 )
 
+        oidc_change_password_url = None
+        if 'OIDC_CHANGE_PASSWORD_URL' in settings:
```

Comment:
Should this be an attribute exist check and not a key exist check?

Reply:
good call thanks @pkdash!